src/components/Pagination/Pagination.css

```
@@ -0,0 +1,19 @@
+.neo-btn-square-secondary--info{
+    background-image: none;
+    background-color: var(--global-background);
+    border-color: var(--neo-color-blue-500);
```

Brayden, you did the right thing here. This is the correct color for the pagination, but, there is something to which you haven't been exposed to yet, which is the way we handle themeing (light mode/dark mode) in Neo.

Basically, our color variables are abstracted one level up -- in this case `--pagination-color` will have the value `blue-500` in light mode, and the corresponding blue color value in dark mode.

We can go over the way this works when I am back from my break, or feel free to reach out to any of the other team members for additional clarification.

In the meantime, please try replacing every occurrence of `--neo-color-blue-500` with `--pagination-color` here as follows:

```suggestion
    border-color: var(--pagination-color);
```

Lmk if that works